docs/userguide/modules/binding-workloads-using-sbo/pages/custom-path-injection.adoc

```diff
@@ -162,3 +162,83 @@ spec:
 ...
 ----
 <1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
+
+
+== Workload resource mapping
+
+[NOTE]
+====
+* Workload resource mapping is available for the secondary workloads of the ServiceBinding custom resource (CR) for both the API groups: `binding.operators.coreos.com` and `servicebinding.io`.
+* You must namespace `ClusterWorkloadResourceMapping` resources under the `servicebinding.io` API group.  However, the `ClusterWorkloadResourceMapping` resources interact with `ServiceBinding` resources under both the `binding.operators.coreos.com` and `servicebinding.io` API groups.
```

Comment:
```suggestion
* You must namespace `ClusterWorkloadResourceMapping` resources under the `servicebinding.io` API group. However, the `ClusterWorkloadResourceMapping` resources interact with `ServiceBinding` resources under both the `binding.operators.coreos.com` and `servicebinding.io` API groups.
```

Reply:
I've migrated to single-spacing between sentences for consistency.

I don't see how point 2 is relevant, since _must_ earlier in the sentence implies no other option is available.  That said, I've made the change.